fiftyone/zoo/models/manifest-torch.json

```diff
@@ -501,7 +501,7 @@
             "base_name": "keypoint-rcnn-resnet50-fpn-coco-torch",
             "base_filename": "keypointrcnn_resnet50_fpn_coco-fc266e95.pth",
             "version": null,
-            "description": "Keypoint R-CNN model from `Keypoint Density-based Region Proposal for Fine-Grained Object Detection and Classification using Regions with Convolutional Neural Network Features <https://arxiv.org/abs/1603.00502>`_ with ResNet-50 FPN backbone trained on COCO",
+            "description": "Keypoint R-CNN model from `Mask R-CNN <https://arxiv.org/abs/1703.06870>`_ with ResNet-50 FPN backbone trained on COCO",
```

Comment:
Just confirming: looks like we were linking to the wrong paper here?

Reply:
Yes believe so, for instance `keypoint-rcnn-resnet50-fpn-coco-torch` on the Pytorch models page also links to Mask R-CNN. It's a bit confusing given the titles but the previously-linked paper doesn't actually predict keypoints (it uses keypts internally for ROI selection to do objdet). Will double-check

Reply:
confirmed previous link was incorrect